src/fetch.js

```diff
@@ -64,6 +64,6 @@ export default (url, opts = {}) => {
             setTimeout(() => attempt(n + 1), delay);
           }
         });
-    }(options.retries));
+    }(0));
```

Comment:
I see `n-1` above -- that will be -1**2, then 0**2.. I think you want to start at 1?

Reply:
🤔 yeah, you're probably right.

Reply:
Revised.